oeis.py

```diff
@@ -247,6 +247,44 @@ def A001220(start: int = 0, limit: int = 2) -> Collection[int]:
     return sequence[start:]
 
 
+@oeis
+def A008587(start: int = 0, limit: int = 20) -> Collection[int]:
+    "Multiples of 5."
+    sequence = []
+
+    for i in range(start, limit):
```

Comment:
Beware, range expects `start`, `stop`, not `start`, `limit`. Also the `stop` argument of `range` is excluded, so `range(10, 12)` is `10, 11`.